specs/consensus/quint/driver.qnt

```diff
@@ -653,9 +653,6 @@ module driver {
             val newstate = { ...state, incomingVotes: state.incomingVotes.exclude(Set(vote))}
             (newstate, VoteDInput(vote))
 
-        else if (state.existsTimeout())
```

Comment:
The comment at the begging of this method should be updated, as it mentions timeouts.